content/docs/install/multi-cluster/index.md

```diff
@@ -163,6 +163,13 @@ The important once that are used in the above example are:
 | `remoteControlPlane.api.token` | Keptn API token | `""` |
 
 
+## Known Limitations
+
+Keptn uses the integration name and version as identifier for connecting the integration to the control plane.
```

Comment:
```suggestion
Each integration plane is assigned an identifier based on the name and version of the Keptn integration service that runs there.
Keptn uses this identifier to connect the Execution Plane to the Control Plane.
```
I'm just guessing at this wording.  The problem I see is that the rest of this section talks about the Execution Plane.  I think you are inferring that the Execution Plane identifier is based on the name of the Keptn service/integration that is installed there.  Maybe this little tidbit should be added to the bullet list at the top of the page that discusses the Execution Plane.  Just need to make the terminology consistent through the page.  It does seem that we may need to add some additional terms to the discussion.

What if one runs multiple Keptn services on an Execution Plane?  Is this identifier formed based on the name of the first service that is installed?

What happens if one runs multiple